src/test/java/bio/terra/stairway/fixtures/TestUtil.java

```diff
@@ -18,17 +18,18 @@
 import java.util.concurrent.TimeUnit;
 import javax.sql.DataSource;
 import org.apache.commons.dbcp2.BasicDataSource;
+import org.apache.commons.lang3.StringUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public final class TestUtil {
-  private static Logger logger = LoggerFactory.getLogger(TestUtil.class);
+  private static final Logger logger = LoggerFactory.getLogger(TestUtil.class);
 
   private TestUtil() {}
 
-  public static final Integer intValue = Integer.valueOf(22);
+  public static final Integer intValue = 22;
```

Comment:
The types don't need to be boxed here
```suggestion
  public static final int intValue = 22;
```